core-web/libs/portlets/edit-ema/portlet/src/lib/shared/models.ts

```diff
@@ -188,3 +188,11 @@ export interface DotDeviceWithIcon extends DotDevice {
 }
 
 export type CommonErrorsInfo = Record<CommonErrors, InfoPage>;
+
+export interface DialogAction {
+    event: CustomEvent;
+    payload: ActionPayload;
+    dirty: boolean;
+    saved: boolean;
+    isTranslation: boolean;
```

Comment:
Can you explain me what each of these booleans do?

Reply:
They dont exist anymore